### PR TITLE
update script of run CI for external PRs to add "Big Models"

### PR DESCRIPTION
**Pull request by fs-eire**
### Description
update script of run CI for external PRs to add "Big Models"

### CODE REVIEW

**Review by tianleiwu — APPROVED** [review]

File: tools/python/run_CIs_for_external_pr.py [hunk 1]
```diff
@@ -93,6 +93,8 @@ def main():
         # checks
         "onnxruntime-python-checks-ci-pipeline",
         "onnxruntime-binary-size-checks-ci-pipeline",
+        # big models
+        "Big Models",
```
Comment (skottmckay) [inline tools/python/run_CIs_for_external_pr.py:97]:
```suggestion
        # other
        "Big Models",
        "Python format",
```

Seems to be another required CI

Comment (tianleiwu) [reply tools/python/run_CIs_for_external_pr.py:97]: "Python format" will run automatically when you click "Approve and run" button following "5 workflows awaiting approval" message in an external PR.

Comment (skottmckay) [reply tools/python/run_CIs_for_external_pr.py:97]: Ah ok. thanks for the info.

Comment (skottmckay) [reply tools/python/run_CIs_for_external_pr.py:97]: That 'Approve and run' doesn't seem to be consistent though. Not showing up on #19079 which is why I thought 'Python format' might need to be added here.

Comment (fs-eire) [reply tools/python/run_CIs_for_external_pr.py:97]: "Python format" is not in Azure Pipelines. It is github Action so I think adding it here would not take effect.

**Pull request merged**